docs/source/reference-core.rst

```diff
@@ -878,9 +881,9 @@ Nursery objects provide the following interface:
       The conventional way to define ``async_fn`` is like::
 
          async def async_fn(arg1, arg2, *, task_status=trio.TASK_STATUS_IGNORED):
-             ...
+             ... # some work before the parent may proceed
```

Comment:
A task with `task_status=trio.TASK_STATUS_IGNORED` can also be started with `start_soon()`, in which case the parts before and after `task_status.started()` behave the same as each other. Is there a different way to word this comment that might be clearer about that?

Reply:
reworded to "some work the parent might wait for"